python/cudf/cudf/core/udf/pipeline.py

```diff
@@ -16,7 +18,48 @@
 precompiled: cachetools.LRUCache = cachetools.LRUCache(maxsize=32)
 
 
-def get_frame_row_type(fr):
+class _FrameJitMetadata(object):
```

Comment:
Python 3 classes do not need to inherit from `object`. ([reference](https://stackoverflow.com/questions/4015417/why-do-python-classes-inherit-object))
```suggestion
class _FrameJitMetadata:
```